action.yml

```diff
@@ -41,5 +41,5 @@ runs:
         path: "~/.cache/pypoetry/virtualenvs/"
         key: "${{ runner.os }}-poetry-${{ steps.python-setup.outputs.python-version }}-${{ hashFiles('./poetry.lock') }}${{ inputs.cache-version }}"
     - name: "Install environment"
-      run: "poetry install"
+      run: "poetry install --only=dev"
```

Comment:
Based on docs it should be `--only dev` I think:
https://python-poetry.org/docs/cli/#install

Also, would it make sense to add this as input?
```
inputs:
  poetry-install-options:
    description: "Options pass to the poetry install command"
    required: false
    default: "--only dev"
```

Then in the install step:
```
    - name: "Install environment"
      run: "poetry install ${{ inputs.poetry-install-options }}"
```